pandas/tests/io/parser/test_skiprows.py

```diff
@@ -14,6 +14,8 @@
 from pandas import DataFrame, Index
 import pandas._testing as tm
 
+pytestmark = pytest.mark.usefixtures("pyarrow_skip")
```

Comment:
@arw2019, I remember some of these tests used to pass while I was still working on the arrow csv reader. Can you make sure nothing broke during your refactors? Thanks.

Reply:
I skipped the tests to try to diagnose the timeouts on CI jobs. Agreed it's better to xfail if possible but currently multiple CI jobs are failing so aiming to get to green then look at re-activating these

Reply:
I recall spending some time trying to get the full test suite to pass for the original PR. IIRC tests that passed in isolation would hang when run as part of the full test run.

so while ideally the tests should be xfailed, unless we can isolate the cause of the lockups, then skips may unfortunately be necessary.